crates/orderbook/src/api.rs

```diff
@@ -28,6 +28,7 @@ mod get_order_by_uid;
 mod get_order_status;
 mod get_orders_by_tx;
 mod get_solver_competition;
+mod get_solver_competition_v2;
```

Comment:
Nice. A v2 version prevents breaking changes for third-parties using the old endpoint.

But shouldn't we also update the OrderBook's `openapi.yml` definition with the new v2 endpoints?